doc-content/enterprise-only/springboot/bus-app-upgrade-proc.adoc

```diff
@@ -0,0 +1,21 @@
+[id='bus-app-upgrade_{context}']
+= Upgrading a business application
+
+If you have a business application that was created for {PRODUCT} 7.1 or 7.2, you must upgrade the Spring Boot version in the business application to version 2.1.1 before you use that application with {PRODUCT} {PRODUCT_VERSION}.
+
+.Prerequisites
+* A business application that was created for {PRODUCT} 7.1 or 7.2 exists.
+
+.Procedure
+. Open the business application `pom.xml` file in a text editor.
+. Find the following element and upgrade the `version` attribute to 2.1.1:
```

Comment:
would be also good to mention to upgrade version of kie.version to be RHPAM 7.3

Reply:
Thanks. Would you please give me an example, similar to that you did for the spring boot version, @mswiderski ?

Reply:
in the same pom.xml as spring version there is a section with properties
```
<properties>
    <version.org.kie>7.13.0.Final</version.org.kie>
```

so that version.org.kie needs to be set to version number that represents 7.3